src/test/java/POM/pages/AlbumsPage.java

```diff
@@ -30,6 +34,12 @@ public AlbumsPage clickArtist(){
     }
     public AlbumsPage clickHeartBtnPluto(){
         wait.until(ExpectedConditions.elementToBeClickable(heartBtnPluto)).click();
+//        heartBtnPluto.click();
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.